# 2D-Maps-for-Ig-domains-and-for-RBDs
 
![Logo](https://github.com/ByrumLab/2D-Maps-for-Ig-domains-and-for-RBDs/blob/main/logo.png)


Hackathon team. Lead: Philippe Youkharibache. Tech: Matteo Manfredi and Chiragkumar Patel. Writer: Stephanie Byrum

# 1. What is the problem? 

**Understanding 3D protein structure within 2D space.** 1D protein sequence provides amino acid residue information; however, critical topological information is completely absent. On the other hand, 3D structure provides toplogical annotation and visualization of domain interactions, but without the amino acid residue information. 2D visualizations provide both sequence information and 3D structual annotations, which allows biologists to quickly interpret the function. 

# 2. Why is it significant? 

Ig domains are an important component of the immune system. They recognize foreign antigens, bind the receptor binding domain (RBD), and clear antigens from the biological system. IgV domains are particularly important due to the ability of the variable region to bind new antigens and RBDs. Visualizing the domain folds and RBD binding in a 2D map will provide insight into regions of gaps or mutations preventing the immune system from recognizing pathogens. 2D maps will provide a tool to identify new therapeutic targets. 

![Graphical_significance](https://github.com/ByrumLab/2D-Maps-for-Ig-domains-and-for-RBDs/blob/main/2Dmap_IgV2D.001.png)

# 3. How to use 2DproteoMapper?

**Workflow** 

**Installation and Requirements** 
